src/plugins/intel_cpu/src/config.cpp

```diff
@@ -252,6 +252,12 @@ void Config::readProperties(const std::map<std::string, std::string> &prop) {
         } else {
             inferencePrecision = ov::element::f32;
         }
+#if defined(OV_CPU_ARM_ENABLE_FP16)
+        //fp16 precision is used as default precision on ARM for non-convolution networks
```

Comment:
Could you please add a justification here

Reply:
I run convolution network in f16/f32 precision and found that `gemm_acl_FP16` is much slower than `gemm_acl_FP32`
Reorder is also slower: fp32 uses `jit_uni_FP32` and fp16 uses `ref_any_FP16`